packages/autocomplete-js/src/__tests__/reverseHighlightHit.test.ts

```diff
@@ -19,7 +19,23 @@ describe('reverseHighlightHit', () => {
         },
         attribute: 'query',
       })
-    ).toMatchInlineSnapshot(`"<mark>amazon </mark>fire tablet<mark>s</mark>"`);
+    ).toEqual([
+      expect.objectContaining({
```

Comment:
This is a simplified representation of a VNode.

Reply:
could we use a serializer and test using a snapshot?

Reply:
We could but I wanted to wait a bit.